seasmon_xr/ops/whit.py

```diff
@@ -110,51 +140,42 @@ def ws2dpgu(y, lmda, nodata, p, out):
     Returns:
         Smoothed time-series array z
     """
-    w = numpy.zeros(y.shape, dtype=float64)
-    m = y.shape[0]
-    n = 0
-
-    for ii in range(m):
-        if y[ii] == nodata:
-            w[ii] = 0
-        else:
-            n += 1
-            w[ii] = 1
+    if lmda != 0.0:
```

Comment:
I added this test in the whit/ws2dpgu function because it was already there in ops/ws2dpgu.py and not up to date in the whit one. But this check isn't from me

Reply:
https://github.com/WFP-VAM/seasmon_xr/blob/main/seasmon_xr/ops/ws2dpgu.py